dump-pe/main.cpp

```diff
@@ -272,15 +272,39 @@ int printSecs(void *N,
   return 0;
 }
 
+int printTls(void *N, VA funcAddr) {
+  static_cast<void>(N);
+  auto address = static_cast<std::uint64_t>(funcAddr);
+  std::cout << "TLS: 0x";
+  std::cout << to_string<decltype(address)>(address, std::hex);
```

Comment:
Had to remove `to_string` since `std::to_string` doesn't understand `std::hex` as a second parameter. The VS2017 compiler didn't know which overload to pick from. I am under the impression something significant changed between `to_string` and `std::to_string`.